crates/hc_sandbox/tests/cli.rs

```diff
@@ -602,12 +627,42 @@ async fn get_created_conductor_config(process: &mut Child) -> ConductorConfig {
             let config = read_config(PathBuf::from_str(config_root_path).unwrap().into())
                 .unwrap()
                 .unwrap();
+
+            tokio::task::spawn(async move {
+                while let Ok(Some(line)) = lines.next_line().await {
+                    println!("@@@-{line}-@@@");
+                }
+            });
+
             return config;
         }
     }
     panic!("getting created conductor config failed");
 }
 
+async fn shutdown_sandbox(mut child: Child) {
```

Comment:
Best effort to shut down the sandbox. This does result in `holochain` being shutdown correctly, as least on Ubuntu. That's not perfect but it is an improvement. There is an equivalent for Windows but it's not so simple. Just dealing with Linux/macos for this PR.

Reply:
Windows was having issues because of `stdout` being kept open by the Tokio task I added. I don't get why, although I'd like to, it's just adding a bit of extra information for debugging tests and we don't develop on Windows so it doesn't seem valuable to figure out. I gave up on trying to send signals on Windows. It's fraught with problems because you have to fiddle with signal handling for the parent process which is the shared test process... so it makes a right mess and sometimes ends up just killing all the tests when the first on finishes. Not worth having that kind of code.